document/cloud-client/parse_key_value.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC. All Rights Reserved.
```

Comment:
License should just be: `# Copyright 2019 Google LLC`

Without the `.` period after LLC
And drop the `All Rights Reserved.`

Reply:
done.